### DISCUSSION TITLE
privileged - problem

### DISCUSSION THREAD

**Discussion by seler69** [blakeblackshear/frigate#9346] [General]
Hello everyone!

What have i done?! :
1. Ubuntu Joshua Riek rk3588 preinstaled - done
2.  Home assistant supervised - done
3. Frigate added from add-ons store - done
i have some issues with my frigate.yml (GPU not working)
i checked (pirveledge status)
for i in $(docker container stats --no-stream| cut -d " " -f 1| grep -v CONTAIN); do docker inspect --format='{{.HostConfig.Privileged}}'  $i; done
only Homeassistant and HA supervised get --priveledge, Frigate not!

is there any way to stop Docker and find proper configuration.yaml and add "privileged: true" like in docker compose and start docker again?

im docker newbie, i made some research but haven't find any tips

**Comment by NickM-27** [comment]
you need to use the full access addon and disable protected mode
